src/data/markdown/docs/30 xk6-browser/01 Get started/04 Module Properties.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Module Properties'
```

Comment:
This seems docs about the API. Don't know how but I'd place it within the Module section.

<img width="274" alt="Screenshot 2022-10-17 at 13 12 51" src="https://user-images.githubusercontent.com/825430/196163133-18ec9359-7f8e-4427-9974-8764cd80be54.png">

Reply:
I've created 
<img width="271" alt="Screenshot 2022-10-17 at 17 56 14" src="https://user-images.githubusercontent.com/10811379/196237818-98c3da58-5f4d-47ae-8c14-96f0a4b2873b.png">